lib/Doctrine/Record.php

```diff
@@ -1815,6 +1816,13 @@ public function getPrepared(array $array = array())
     {
         $a = array();
 
+        /**
+         * This was previously unset (unless the if condition below is true),
+         * setting to empty array, which won't change behavior.
+         * @todo It seems like this is meant to be set to what's in $array, but not sure
+         */
+        $modifiedFields = array();
```

Comment:
As the comment says, I think this should be set to what's in `$array` (passed into method), but I didn't want to alter the behavior here.